### PR TITLE
Move some code into the open

### PR DESCRIPTION
**Pull request by jaredpar**

### CODE REVIEW

**Comment by jaredpar** [comment]
CC @Pilchie, @jasonmalinowski,

**Comment by jaredpar** [comment]
I will delete the closed project after I merge.

**Comment by Pilchie** [comment]
Happy to see this, but @heejaechang should probably also see it.

**Comment by heejaechang** [comment]
nice! :+1:

File: Restore.cmd [hunk 1]
```diff
@@ -7,6 +7,7 @@ set NuGetAdditionalCommandLineArgs=-verbosity quiet -configfile "%RoslynRoot%nug
 :ParseArguments
 if /I "%1" == "/?" goto :Usage
 if /I "%1" == "/clean" set RestoreClean=true&&shift&& goto :ParseArguments
+if /I "%1" == "/fast" set RestoreFast=true&&shift&& goto :ParseArguments
```
Comment (jasonmalinowski) [inline Restore.cmd]: Why the /fast flag? There's correctness reasons for doing the deletions, it's not just paranoia.

Comment (jaredpar) [inline Restore.cmd]: It's for iterating locally a bit faster.  I'm having to restore a lot during this change and full restore was taking up too much time.  Needed to trim it down. 

I realize it's unsafe and shouldn't be done in a real build environment.  For local iteration though I'm willing to absorb some risk.

**Pull request merged**